### DISCUSSION TITLE
Problemi scrutinio

### DISCUSSION THREAD

**Discussion by mpignatelli** [iisgiua/giuaschool#554] [Richieste di aiuto]
Salve ho un problema con l'avvio dello scrutinio degli esami finali. Come devo fare 
![Screenshot_2023-07-28-16-21-38-07_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/iisgiua/giuaschool/assets/138996613/21104495-26c7-4511-8333-68c4ffa7da4a)

**Comment by trinko** [comment]
Duplicato

**Reply by mpignatelli** [reply]
In che senso? Duplicato

**Reply by mpignatelli** [reply]
Mi invia un'email a questo indirizzo così le invio le credenziali: michelina.pignatelli1983@gmail.com

**Comment by trinko** [comment]
E' duplicato perché le segnalazioni vanno inserite solo nelle ISSUES non nelle discussioni.

Quanto all'email, io non faccio un servizio di assistenza personale.
